openapi.yaml

```diff
@@ -8025,10 +8025,10 @@
             "type" : "integer",
             "format" : "int32"
           },
-          "highestEnv" : {
+          "topicOwner" : {
             "type" : "boolean"
           },
-          "topicOwner" : {
+          "highestEnv" : {
             "type" : "boolean"
           }
```

Comment:
These things make me crazy since it is changing all the time ()I mean just the order ....
i there a way to force the order in this file for build on any system?

Reply:
@snuyanzin  yea the variables seems to be incorrectly created. [here](https://github.com/Aiven-Open/klaw/blob/63c4344b1447e88aadfe5aca84b5524f5e3a1720/core/src/main/java/io/aiven/klaw/model/TopicOverviewInfo.java#L14)  we can try replacing isTopicOwner with topicOwner and isHighestEnv with just highestEnv